Ark.Tools.FtpClient.ArxOne/FtpClientPoolArxOneWithSocksAndConfigurerFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) 2023 Ark Energy S.r.l. All rights reserved.
+// Licensed under the MIT License. See LICENSE file for license information. 
+using Ark.Tools.FtpClient.Core;
+using EnsureThat;
+using System;
+using ArxOne.Ftp;
+
+namespace Ark.Tools.FtpClient
+{
+    public class FtpClientPoolArxOneWithSocksAndConfigurerFactory : IFtpClientPoolFactory
+    {
+        private readonly ISocksConfig _config;
+        private readonly Action<FtpClientParameters> _configurer;
+
+        public FtpClientPoolArxOneWithSocksAndConfigurerFactory(ISocksConfig config, Action<FtpClientParameters> configurer)
```

Comment:
No, the new Factory should have only 1 Config property in the ctor to avoid this issue again if we have to expand.

Name should no longer be tied with how many paramters tbe Config supports.